src/include/connection.h

```diff
@@ -309,8 +309,17 @@ struct __wt_connection_impl {
 	bool		 evict_server_running;/* Eviction server operating */
 
 	WT_THREAD_GROUP  evict_threads;
-	uint32_t	 evict_threads_max;/* Max eviction threads */
-	uint32_t	 evict_threads_min;/* Min eviction threads */
+	uint32_t         evict_threads_max;/* Max eviction threads */
```

Comment:
Did you consider adding these fields to the ``WT_CACHE`` structure instead of ``WT_CONNECTION_IMPL``?